### PR TITLE
Copy student_notes load test from the old repo

### PR DESCRIPTION
**Pull request by pwnage101**
This PR copies the student_notes load test from the deprecated load-tests repo (first commit) then modernizes it (second commit).

Commit messages:

```
commit 8c1be67d8d050965fec9a7e6aa8ec8c627f77f09
Author: Troy Sankey <sankeytms@gmail.com>
Date:   Wed May 24 14:45:02 2017 -0400

    [student_notes] modernize this load test

    * move EdxAppTasks out of the lms test and into helpers
    * course enrollment support
    * settings file support
    * correct import style
    * correct __init__.py
    * add requrements.txt
    * other adjustments which fix code making old assumptions (circa 2015)

commit fe4c09abc25a245126636095307b6ce518f2c476
Author: Troy Sankey <sankeytms@gmail.com>
Date:   Wed May 24 14:55:41 2017 -0400

    copy student_notes loadtest from the old repo

    * from https://github.com/edx/load-tests/tree/8462f27b/locust/student_notes
```

The changes to the LMS test should result in a no-op, but there is a slight chance of a regression.

Testing the student_notes load test for runnability: https://build.testeng.edx.org/job/loadtest-driver/61/console
Testing the lms load test for regressions: https://build.testeng.edx.org/job/loadtest-driver/62/console

Obviously there's room for improvement, but the student_notes load test is basically runnable.

### CODE REVIEW

**Comment by pwnage101** [comment]
@MichaelRoytman please review.

@clintonb can you review?  In particular i'm moving more common things out of lms load test and into the helpers directory.

**Comment by pwnage101** [comment]
Reviewers: it helps to only look at the second commit, not the aggregate you would get when you click on the "Files changed" tab.

File: loadtests/student_notes/locustfile.py [hunk 1]
```diff
@@ -68,15 +74,16 @@ def __init__(self, *args, **kwargs):
...
+        self.enroll(self.course_id)
 
     @property
     def annotator_auth_token(self):
         """Get the JWT key for making requests to the notes service from the LMS."""
         return self.client.get(
             '/courses/{course_id}/edxnotes/token/'.format(course_id=self.course_id),
             headers={'content-type': 'text/plain'},
-            verify=False  # Skip SSL verification
+            # verify=False  # Skip SSL verification
```
Comment (clintonb) [inline loadtests/student_notes/locustfile.py]: Avoid commented-out code. Delete this if it's not used.

File: loadtests/student_notes/locustfile.py [hunk 2]
```diff
@@ -110,14 +117,17 @@ def _request_from_notes_service(self, method, path, params_or_body=None, **kwarg
                 'X-CSRFToken': self.client.cookies.get('csrftoken', ''),
                 'content-type': 'application/json',
             },
-            'verify': False  # Skips SSL verification
+            # 'verify': False  # Skips SSL verification
```
Comment (clintonb) [inline loadtests/student_notes/locustfile.py]: Same as above.

File: loadtests/student_notes/locustfile.py [hunk 3]
```diff
@@ -144,10 +154,16 @@ def _create_note(self):
         data = {
             "user": self._anonymous_user_id,
             "course_id": self.course_id,
-            "text": ' '.join(pick_some(NOTES_TEXT, NUM_WORDS)),
-            "tags": pick_some(NOTES_TEXT, NUM_TAGS),
+            "text": ' '.join(pick_some(
```
Comment (clintonb) [inline loadtests/student_notes/locustfile.py]: Nit: pick a quote style (single or double) and use it throughout.

**Comment by clintonb** [comment]
Remember to delete your merged branches.

**Pull request merged**